build.sbt

```diff
@@ -115,7 +115,8 @@ lazy val root = Project(id="commons", base=file("."))
   .settings(description := "Scala commons for Fulcrum Genomics.")
   .settings(
     libraryDependencies ++= Seq(
-      "org.scala-lang" %  "scala-reflect" %  scalaVersion.value,
+      "com.typesafe"   %  "config"        % "1.3.2",
```

Comment:
I guess this is fine, but I think we should be really careful about adding dependencies to `commons`.  It's a small project of utility classes, and even in this case the dependencies are now larger than the commons.jar we produce :/